crates/ibc/src/core/ics03_connection/handler/conn_open_confirm.rs

```diff
@@ -39,15 +39,17 @@ pub(crate) fn process(
         conn_end.delay_period(),
     );
 
-    // 2. Pass the details to the verification function.
-    verify_proofs(
+    verify::verify_connection_proof(
```

Comment:
Unrelated to this PR, but we're missing `validateSelfClient()`. Will open an issue for this if there already isn't one.

Reply:
I don't think we have an issue for that yet indeed